byway/src/main/webapp/script.js

```diff
@@ -26,3 +26,56 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
```

Comment:
I'd delete these functions since we're not using them (from the portfolio stuff)

Reply:
done.